Sources/SwiftSyntaxBuilder/ArrayElementListConvenienceInitializers.swift

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+extension ArrayElementList: ExpressibleAsArrayExpr {
```

Comment:
Do we really need to hand-craft a convenience initializer here? Can’t we just add this to `ExpressibleAsConformances`?